presto-cli/pom.xml

```diff
@@ -88,7 +89,7 @@
 
         <dependency>
             <groupId>org.jline</groupId>
-            <artifactId>jline-terminal-jna</artifactId>
+            <artifactId>jline-terminal-jansi</artifactId>
```

Comment:
We want to move away from Jansi. See #2878

Reply:
I couldn't get JNA working on native-image (that's why I've switched to JANSI)

Reply:
Solved.